templates/vhost/_directories.erb

```diff
@@ -305,6 +305,66 @@
     <%- if directory['auth_ldap_group_attribute_is_dn'] == 'on' -%>
     AuthLDAPGroupAttributeIsDN On
     <%- end -%>
+    <%- if directory['auth_ldap_initial_bind_pattern'] -%>
+    AuthLDAPInitialBindPattern <%= directory['auth_ldap_initial_bind_pattern'] %>
+    <%- end -%>
+    <%- if directory['auth_ldap_bind_as_user'] == 'off' -%>
+    AuthLDAPInitialBindAsUser Off
+    <%- end -%>
```

Comment:
Would it be better to use this pattern for booleans?
```suggestion
    <%- unless directory['auth_ldap_bind_as_user'].nil? -%>
    AuthLDAPInitialBindAsUser <%= scope.call_function('apache::bool2httpd', [directory['auth_ldap_bind_as_user']]) %>
    <%- end -%>
```

Reply:
I just followed the pattern in the file, and I agree it would be lovely if we changed everything to a single call. Should this be a separate PR?